exercises/mongo-shell/basic-mongo-queries.md

```diff
@@ -1,7 +1,157 @@
-## READ Operations
+# CRUD Operations
+
+This document deals with basic create, read, update and delete operations using the mongo shell. 
```

Comment:
```suggestion
This document deals with basic **Create**, **Read**, **Update** and **Delete** operations using the mongo shell. 

```


I think distinction that these are the 'types' of operations is helpful when you are new to acronyms.